pkg/ocm/ocm.go

```diff
@@ -57,51 +55,48 @@ type SdkClient struct {
 
 // New will create a new ocm client by using the path to a config file
 // if no path is provided, it will assume it in the default path
-func New(ocmConfigFile string) (*SdkClient, error) {
+func NewFromClientKeyPair(ocmURL string, clientID string, clientSecret string) (*SdkClient, error) {
 	var err error
 	client := SdkClient{}
 
-	// The debug environment variable ensures that we will never use
-	// an ocm config file on a cluster deployment. The debug environment variable
-	// is only for local cadctl development
-	debugMode := os.Getenv("CAD_DEBUG")
```

Comment:
```suggestion
	// The debug environment variable ensures that we will never use
	// an ocm config file on a cluster deployment. The debug environment variable
	// is only for local cadctl development
	debugMode := os.Getenv("CAD_DEBUG")
```
Probably a good idea to keep at least this comment somewhere, because I initially expected the `CAD_DEBUG` to be more something along the lines of a debug-level logging setting, not something that changes behaviour that much.